types/three/three-core.d.ts

```diff
@@ -6065,6 +6051,26 @@ export class WebGLLights {
     get(light: any): any;
 }
 
+/** 
+ * An object with a series of statistical information about the graphics board memory and the rendering process. 
+ */ 
+export class WebGLInfo { 
+    autoReset: boolean; 
+    memory: { 
+        geometries: number; 
+        textures: number; 
+    }; 
+    programs: number | null; 
+    render: { 
+        calls: number; 
+        lines: number; 
+        points: number; 
+        triangles: number; 
+    }; 
+    reset(): void; 
+    update(): void; 
```

Comment:
update takes params, but is this part of three's intentionally exposed API? Is it documented?

Reply:
Correct... well it is exposed with the rest of the object but it's only supposed to be used internally. Removed from the types.